Print_A_Bot/Print_A_Bot/settings.py

```diff
@@ -129,6 +107,15 @@
     },
 ]
 
+CHANNEL_LAYERS = {
```

Comment:
this one matters, to test locally `brew install redis`

and/or take a look at https://medium.com/@petehouston/install-and-config-redis-on-mac-os-x-via-homebrew-eb8df9a4f298 for more notes on redis and

https://channels.readthedocs.io/en/stable/deploying.html
for notes on `asgi_redis`